Cargo.toml

```diff
@@ -1,27 +1,27 @@
 [workspace]
 default-members = ["bin/spiced"]
 members = [
-  "bin/spiced/",
```

Comment:
I don't see any new references, so assume it is just a formatting change, also scanned the changes and they look good.